### PR TITLE
DOCS-15438 fle2 not supported

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCS-15438-fle2-not-supported-v0.9.0/reference/limitations/)

[JIRA](https://jira.mongodb.org/browse/DOCS-15438)

### CODE REVIEW

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -44,10 +44,11 @@ General Limitations
 - Network compression is not supported.
 - :dbcommand:`applyOps` operations from the source cluster are not
   supported.
-- Serverless clusters are not supported.
-- The MongoDB Shared Tier is not supported.
 - The :data:`system.js <<database>.system.js>` collection is not
   replicated.
+- Serverless clusters are not supported.
+- The MongoDB Shared Tier is not supported.
+- :ref:`Queryable Execution <qe-manual-feature-qe>` is not supported.
```
Comment (davemungo) [inline source/reference/limitations.txt:51]: This link doesn't work yet. It will work when the QE docs are pushed in 6.0

**Review by davemungo — COMMENTED** [review]
Note about upcoming link

**Review by jason-price-mongodb — APPROVED** [review]
@davemungo LGTM. Cheers.

**Review by XueruiFa — APPROVED** [review]
LGTM

**Pull request merged**